.github/CHANGELOG.md

```diff
@@ -46,6 +46,9 @@
 
 ### Improvements
 
+*  Fix PTM stable latest related to the default.qubit.legacy. 
```

Comment:
```suggestion
*  Fix PTM stable latest related to `default.qubit.legacy`. 
```

Reply:
Shouldn't this be listed under "Bug fixes"?

Reply:
You are right! I will move it and disable all labels for a quick merge after rerunning the minimum test batch.